public/app/core/utils/explore.ts

```diff
@@ -395,3 +385,11 @@ export const getQueryKeys = (queries: DataQuery[], datasourceInstance: DataSourc
 
   return queryKeys;
 };
+
+export const getTimeRange = (timeZone: TimeZone, rawRange: RawTimeRange): TimeRange => {
+  return {
+    from: dateMath.parse(rawRange.from, false, timeZone.raw as any),
```

Comment:
If we extend `Timezone` type there is no need for `as any` here
```suggestion
    from: dateMath.parse(rawRange.from, false, timeZone.raw),
```

Reply:
Decided to solve this later since it requires a lot of changes. Updated #16806